misk/web/tabs/web-actions/src/containers/WebActionsContainer.tsx

```diff
@@ -120,15 +120,27 @@ const Metadata = (
   props: {
     content: string | JSX.Element
     label: string
-    tooltip: string | JSX.Element
+    tooltip?: string | JSX.Element
   } & any
-) => (
-  <MenuItem
-    label={props.label}
-    text={<Tooltip content={props.tooltip}>{props.content}</Tooltip>}
-    {...props}
-  />
-)
+) => {
+  if (props.tooltip) {
+    return (
+      <MenuItem
+        label={props.label}
+        text={<Tooltip content={props.tooltip}>{props.content}</Tooltip>}
+        {...props}
+      />
+    )
+  } else {
+    return (
+      <MenuItem
+        label={props.label}
+        text={<Tooltip>{props.content}</Tooltip>}
```

Comment:
Probably misunderstanding but do we want the `Tooltip` tag here?

Reply:
An empty one provides identical padding and margin of the element.